docs/docs/_index.md

```diff
@@ -77,7 +77,7 @@ spec:
 
 ## Cross namespace grafana instances
 
-As described in [#44](https://github.com/grafana-operator/grafana-operator-experimental/issues/44) we didn't want it
+As described in [#44](https://github.com/grafana/grafana-operator-experimental/issues/44) we didn't want it
```

Comment:
The other repo hasn't been migrated, so it won't work.
We could rewrite this section to just take the content of that issue. Or we can just keep the old link